FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -41,9 +40,9 @@ def main():
     Globals.debug_recorder = app_tools.DebugRecorder(debug_file)
 
     # Initialize a festival data object.
-    iffr_data = IffrData(plandata_dir)
+    iffr_data: IffrData = IffrData(plandata_dir)
```

Comment:
Please explain what this is for.

Reply:
I started using PyCharm, and it suggested to use `type hinting` at this point.
Never too old to learn, I found this a good occasion to start using it.